pkg/api/v1/types.go

```diff
@@ -311,6 +313,8 @@ type PersistentVolumeSource struct {
 	FC *FCVolumeSource `json:"fc,omitempty"`
 	// Flocker represents a Flocker volume attached to a kubelet's host machine and exposed to the pod for its usage. This depends on the Flocker control service being running
 	Flocker *FlockerVolumeSource `json:"flocker,omitempty"`
+	// AzureFile represents an Azure File Service mount on the host that shares a pod's lifetime
+	AzureFile *AzureFileVolumeSource `json:"azureFile,omitempty"`
```

Comment:
This seems like it should be called `AzureFileService` and the volume source should be called `AzureFileServiceVolumeSource`